docs/tutorial/openapi-callbacks.md

```diff
@@ -0,0 +1,16 @@
+There are cases where the API that is being developed will require the clients to implement callback handlers, that will in turn be invoked
+by the application. 
+
+Since this is a cuommon usecase, the OpenAPI spec defines how these callbacks should be documented: <a href="https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.2.md#callbackObject">OpenAPI callback object spec</a>,
```

Comment:
```suggestion
Since this is a common usecase, the OpenAPI spec defines how these callbacks should be documented: <a href="https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.2.md#callbackObject">OpenAPI callback object spec</a>,
```